tuning/automation/rocblas-parser.py

```diff
@@ -193,9 +116,13 @@ def readfile(filename):
         if not (gemmtype in matrices):

```

Comment:
I think it is better to skip the following code if gemmtype is not valid (including "unrecognized").
Maybe better to print some message to notify unsupported types?